src/tools/status_tool.cpp

```diff
@@ -698,6 +700,40 @@ int StatusTool::PrintChunkserverStatus(bool checkLeftSize) {
     std::map<PoolIdType, std::vector<uint64_t>> poolChunkLeftSize;
     std::map<PoolIdType, std::vector<uint64_t>> poolWalSegmentLeftSize;
     std::vector<ChunkServerIdType> offlineCs;
+    bool useWalPool = false;
+    bool useChunkFilePoolAsWalPool = true;
+
+    if (!poolChunkservers.empty()) {
+        ChunkServerInfo chunkserver = poolChunkservers.begin()->second[0];
```

Comment:
Only query the first chunkserver? Why?

Reply:
> Only query the first chunkserver? Why?

Because all chunkserver configurations are the same